### PR TITLE
Feat/serch bar

### PR DESCRIPTION
**Pull request by MacenaAndre**
- Created getUserByName in axios.js
- Created UseEffect to API requisition
- Layout adjustments

### CODE REVIEW

**Review by Francksuel — APPROVED** [review]
Great job!

File: src/components/commons/Navbar.js [hunk 1]
```diff
@@ -1,16 +1,34 @@
...
   const [search, setSearch] = useState();
+  const [usersSearch, setUsersSearch] = useState([]);
+  const [hiddenSearch, setHiddenSearch] = useState(true);
   const navigate = useNavigate();
 
+  useEffect(() => {
+
+    getUserByName({config, search})
+      .then((res) => {
+        console.log(res.data);
```
Comment (tissyane) [inline src/components/commons/Navbar.js]: Remove console.log

File: src/components/commons/Navbar.js [hunk 2]
```diff
@@ -1,16 +1,34 @@
...
+        setUsersSearch(res.data);
+      })
+      .catch((res) => {
+        console.log(res);
```
Comment (tissyane) [inline src/components/commons/Navbar.js]: Maybe we can have an alert here instead of console.log?

File: src/services/axios.js [hunk 3]
```diff
@@ -34,6 +34,11 @@ function toggleLike(postId, config) {
 	return axios.post(`${API}/likes/${postId}`, {}, config);
 }
 
+function getUserByName({ config, search }) {
+  console.log(search);
```
Comment (tissyane) [inline src/services/axios.js]: Remove console.log

**Review by tissyane — APPROVED** [review]
Please see suggestions before merging ;)

**Comment by MacenaAndre** [comment]
Thanks for the comments, all console.logs removed, and .catch responding correctly.

**Pull request merged**